ibc-apps/ics721-nft-transfer/types/src/class.rs

```diff
@@ -144,7 +144,7 @@ impl TryFrom<RawClassTrace> for PrefixedClassId {
         let base_class_id = ClassId::from_str(&value.base_class_id)?;
         // FIXME: separate `TracePath` error.
         let trace_path = TracePath::from_str(&value.path)
```

Comment:
This comment looks redundant.

Reply:
[ed5ed8e](https://github.com/cosmos/ibc-rs/pull/1318/commits/ed5ed8e6a085545c384a1aba36338b6ef1380dff)